plugins/braintree-payment/src/providers/payment-braintree/src/core/braintree-base.ts

```diff
@@ -107,6 +108,8 @@ class BraintreeBase extends AbstractPaymentProvider<BraintreeOptions> {
         publicKey: this.options_.publicKey!,
         privateKey: this.options_.privateKey!,
       });
+      const tokenData = await this.gateway.clientToken.generate({});
```

Comment:
wait... won't this give all users the same clientToken??

Reply:
ok cool. but we'll work around it .. its simple.. we'll just cache on per customer basis.. instead of loading into a constructor. So only the first add to cart will be slow..